toolkit/components/uniffi-bindgen-gecko-js/src/templates/js/Records.jsm

```diff
@@ -0,0 +1,44 @@
+
+{%- for record in ci.iter_record_definitions() %}
+
+class {{ record.js_name() }} {
+    constructor({{ record.js_constructor_field_list() }}) {
+        {%- for field in record.fields() %}
+        this.{{field.js_name()}} = {{ field.js_name() }};
+        {%- endfor %}
+    }
+}
+
+class FfiConverter{{ record.js_name() }} {
+    static lift(buf) {
+        return this.read(new ArrayBufferDataStream(buf));
+    }
+    static lower({{ record.js_var_name() }}) {
```

Comment:
Could this just be a generic argument name like `value`?  If so, then I think we could remove the `js_var_name()` method.